spec/features/user_sees_totals_per_person_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "user sees totals per person" do
+  scenario "total amounts budgeted and spent on one purchased gift" do
+    user = create(:user)
+    list = create(:list, user: user)
+    person = create(:person, name: "Mrs. Clause", amount: 100, list: list)
+    gift_1 = create(:purchased_gift, name: "Electronics", amount: 50, person: person)
+    visit list_person_path(person.list, person, as: user)
+
+    expect(page).to have_person_name("Mrs. Clause")
+    expect(page).to have_budgeted_total("$100.00")
+    expect(page).to have_spent_total("$50.00")
+  end
+
+  scenario "total amounts budgeted and spent on two purchased gifts" do
+    user = create(:user)
+    list = create(:list, user: user)
+    person = create(:person, name: "Rails Rumble Team", amount: 100, list: list)
+    gift_1 = create(:purchased_gift, name: "Fine China", amount: 50, person: person)
```

Comment:
Useless assignment to variable - `gift_1`. Did you mean `gift_2`?<br>Line is too long. [84/80]